test/algebraic_solver.jl

```diff
@@ -0,0 +1,48 @@
+using Symbolics
+using Test
+using LambertW
+
+
+#Testing
+
+@testset "solving tests" begin
+    function hasFloat(expr)#make sure answer does not contain any strange floats
```

Comment:
`hasfloat`, `correctans`, `has_float`, etc. using camelCase does not match the standard style, see the SciMLStyle guide.